plugins/auto-sizes/includes/auto-sizes.php

```diff
@@ -49,7 +49,7 @@ function auto_sizes_update_image_attributes( $attr ): array {
  */
 function auto_sizes_update_content_img_tag( $html ): string {
 	if ( ! is_string( $html ) ) {
-		$html = '';
+		$html = ''; // @codeCoverageIgnore
```

Comment:
I don't think this should be ignored because technically this is a code path that can be tested.